### PR TITLE
Destination Clickhouse: enable normalization for Secure connections

### PR DESCRIPTION
**Pull request by kzzzr**
## What

Enable normalization for Destination Clickhouse with SSL connection by configuring `Secure` parameter in `profiles.yml`
https://github.com/airbytehq/airbyte/issues/9837

## How

By supplying key `"secure": config["ssl"]` to method `transform_clickhouse`

## Recommended reading order
1. `transform.py`
2. `test_transform_config.py`

## 🚨 User Impact 🚨

No breaking changes

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

- `TestTransformConfig.test_transform_clickhouse`

```
./gradlew :airbyte-integrations:bases:base-normalization:unitTest
```

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
thanks for the contribution @kzzzr is it possible to you sign the CLA? The team will review your contribution during this week.

File: airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py [hunk 1]
```diff
@@ -279,6 +279,7 @@ def transform_clickhouse(config: Dict[str, Any]):
             "port": config["port"],
             "schema": config["database"],
             "user": config["username"],
+            "secure": config["ssl"]
```
Comment (marcosmarxm) [inline airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py:282]: My only concern here is if the config file created by the UI read as boolean and not as a string.

**Review by marcosmarxm — APPROVED** [review]
LGTM only I want to test if the dbt container read correctly the value.

File: airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py (hunk 1 above)
Comment (kzzzr) [reply airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py:282]: Hey, it is boolean:
https://github.com/airbytehq/airbyte/blob/a4e8417ec88d740c3d340b2d1df4f9a69e80117f/airbyte-integrations/connectors/destination-clickhouse/src/main/resources/spec.json#L59-L65

I've managed to run normalization successfully, only I had to change `NORMALIZATION_VERSION = "dev"`, build `SUB_BUILD=PLATFORM ./gradlew build` and test it with UI locally.
https://github.com/airbytehq/airbyte/blob/a4e8417ec88d740c3d340b2d1df4f9a69e80117f/airbyte-workers/src/main/java/io/airbyte/workers/normalization/NormalizationRunnerFactory.java#L17

However I think it was hard way to test this.

Comment (marcosmarxm) [reply airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py:282]: Awesome!

**Comment by marcosmarxm** [comment]
having some problems publishing the new version of normalization.

**Pull request merged**